src/caustics/lenses/base.py

```diff
@@ -129,13 +129,18 @@ def forward_raytrace(
             Tensor of source redshifts.
         params: (Packed, optional)
             Dynamic parameter container for the lens model. Defaults to None.
+
         epsilon: Tensor
             maximum distance between two images (arcsec) before they are considered the same image.
+        *Unit: arcsec*
```

Comment:
The spacing doesn't match the rest of the usages.

Reply:
Updated